presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -1044,6 +1044,14 @@ let QueryDetail = React.createClass({
                                         { query.session.remoteUserAddress }
                                     </td>
                                 </tr>
+                                <tr>
+                                    <td className="info-title">
+                                        Client Tags
+                                    </td>
+                                    <td className="info-text">
+                                        { query.session.clientTags.join(", ") }
```

Comment:
Does this fail if the value is null?

Reply:
Yes, but silently. Also, this can't be null.